plugins/node/opentelemetry-instrumentation-mongodb/src/instrumentation.ts

```diff
@@ -216,8 +268,43 @@ export class MongoDBInstrumentation extends InstrumentationBase<
     };
   }
 
+  /** Creates spans for command operation */
+  private _getV4PatchCommand() {
+    const instrumentation = this;
+    return (original: V4Connection['command']) => {
+      return function patchedV4ServerCommand(
+        this: unknown,
+        ns: any,
+        cmd: any,
+        options: undefined | unknown,
+        callback: any
+      ) {
+        const currentSpan = trace.getSpan(context.active());
+        const resultHandler = callback;
+        if (
+          !currentSpan ||
+          typeof resultHandler !== 'function' ||
+          typeof cmd !== 'object' ||
+          cmd.ismaster
```

Comment:
I think you are trying to filter out [this method](https://docs.mongodb.com/v4.2/reference/method/db.isMaster/)
According to the docs, it is deprecated and the new method is [hello](https://docs.mongodb.com/v4.2/reference/method/db.hello/#db.hello).

[This is the relevant part from the driver](https://github.com/mongodb/node-mongodb-native/blob/dd81438593f9900122ed16a583eb9c84d0408b76/src/cmap/connect.ts#L231):
```js
    [serverApi?.version ? 'hello' : LEGACY_HELLO_COMMAND]: true,
```

[where](https://github.com/mongodb/node-mongodb-native/blob/dd81438593f9900122ed16a583eb9c84d0408b76/src/constants.ts#L122)
```js
export const LEGACY_HELLO_COMMAND = 'ismaster';
```

I suggest adding `hello` as well to the check

Reply:
Good point, Fixed